test/metabase/query_processor/middleware/annotate_test.clj

```diff
@@ -460,8 +460,7 @@
   (qp.store/with-metadata-provider meta/metadata-provider
     (testing (str "if a driver is kind enough to supply us with some information about the `:cols` that come back, we "
                   "should include that information in the results. Their information should be preferred over ours")
-      (is (=? {:cols [{:name           "metric"
-                       :display_name   "Total Events"
+      (is (=? {:cols [{:display_name   "Total Events"
```

Comment:
`metric` is a fallback name appearing here because the metric is not defined at all. I think the point of the test is not to check that `metric` is returned as name for a metric that has no aggregation (such metrics do not exist), so I just removed this check.